prow/plugins/lgtm/lgtm.go

```diff
@@ -142,7 +143,7 @@ func handle(gc githubClient, config *plugins.Configuration, ownersClient repoown
 		if err != nil {
 			return err
 		}
-		if !loadReviewers(ro, filenames).Has(commentAuthor) {
+		if !loadReviewers(ro, filenames).Has(strings.ToLower(commentAuthor)) {
```

Comment:
Please use `github.NormLogin`

Reply:
agh, literally had this comment but it got left pending.
anyhow the other point I had was that perhaps we should rename `NormLogin`, I think in some contexts you'd be more likely to think of it as a "username"? /shrug

Reply:
updated